docs/_partials/_conversationPrefillTextSdkxUnityCode.mdx

```diff
@@ -0,0 +1,21 @@
+Example:
+
+```csharp
+using Helpshift;
+// other imports
+
+public class MyGameControl : MonoBehaviour
+{
+    private HelpshiftSdk help;
+    void Awake() {
+      this.help = HelpshiftSdk.GetInstance();
+      help.Install(appId, domainName);
```

Comment:
@sumeet-helpshift This example adds the config map in the OpenHelpshift function, instead of the Awake function:
https://developers.helpshift.com/sdkx-unity/sdk-configuration-android/#fullprivacy

Are both things valid or should we also update the Full privacy example?

Reply:
My bad, we have default config param as null in the Install function signature.
This code is alright.